setup/ubuntu/bootstrap.sh

```diff
@@ -157,6 +157,10 @@ apt-get install -y libffi-dev libssl-dev
 # MySQL dependencies:
 apt-get install -y libmysqlclient-dev
 
+# Microsoft SQL Server dependencies (`sudo` required):
+sudo apt-get install -y freetds-dev
+sudo pip install pymssql==2.1.1
```

Comment:
No need to add it explicitly as the bootstrap script for Ubuntu install all requirements from `requirements_all_ds.txt`.